qcodes/instrument_drivers/Spectrum/M4i.py

```diff
@@ -611,7 +611,7 @@ def convert_to_voltage(self, data, input_range):
         return data * input_range / resolution
 
     def initialize_channels(self, channels=None, mV_range=1000, input_path=0,
-                            termination=0, coupling=0, compensation=None, memsize=2**12):
+                            termination=0, coupling=0, compensation=None, memsize=2**12, pretrigger_memsize = None):

```

Comment:
Why not make the default value of this argument `16` instead of `None`? This will remove the need for a couple of lines of code below, and simplify the code because `pretrigger_memsize` will become of type `int` instead of `Optional[int]`.

Reply:
@astafan8 I agree. There might be future cases where we want to calculate the actual value, but since that is currently not there let's just use the smallest value possible.